pages/HotMealDayCard.js

```diff
@@ -0,0 +1,46 @@
+import { useRouter } from "next/router";
+
+function HotMealDayCard() {
+  const router = useRouter();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const response = await fetch("/api/create-stripe-session", {
+      body: JSON.stringify({
+        amount: 250 * 100,
+        cause: "Hot Meal for 60 Hungry People.",
```

Comment:
I noticed your thoughtfulness here about the wording for the Stripe checkout page. It's very nice.